homework/homework_part_2/javascript/app.js

```diff
@@ -0,0 +1,103 @@
+//1. With interpolation you apply your variable inside 
+// of a string and concatenation you write everything out 
+// as a string 
+// INTERPOLATION: const red = 2
+//const blue = 5
+//console.log(`Hello World ${red + blue}`)
+//CONCATENATION: console.log("Hello World" +  " AND GOOD DAY");
+
+//2. DRY stands for Don't Repeat Yourself, we should pay 
+// attention to it because we don't want to repeat
+//code that is almost identical to eachother when 
+//we can condense it down
+
+//3. you declare a variable at the top ex: let g =4;
+// you assign when you change the value of in this case g 
+// g = 7;
+// and it can be defined depending where the declaration sits
+// if your decleration is outside a for loop and your console.log
+// is inside it will be undefined
```

Comment:
Careful: 
If you declare outside a `for` loop it will be defined inside the `for` loop. 
However, if you declare inside a `for` loop it will be undefined outside the loop.
This is called "scope" and we will talk about it more today.